.cursor/rules/follow-docs.mdc

```diff
@@ -0,0 +1,51 @@
+---
+description: Documentation adherence for feature development and schema changes
+globs:
+  - '**/*.ts'
+  - '**/*.js'
+  - '**/*.json'
+  - '**/*.md'
+alwaysApply: true
+---
+
+## Documentation Requirements
+
+When adding new features or making schema changes, you must:
+
+1. **Review existing documentation** in `/docs` to understand the current structure and standards
+2. **Follow established patterns** from existing documentation when implementing new features
+3. **Reference relevant documentation** sections when making changes to ensure consistency
+
+## Documentation Structure
+
+The project maintains documentation under `/docs/` with the following structure:
+
+- `/docs/` - Project documentation including guides, API references, and examples
+
+## Feature Development Guidelines
+
+### Before Adding New Features:
+
+1. Check if documentation exists in `/docs/` for similar features
+2. Review any relevant schemas and type definitions in the codebase
+3. Understand the overall project structure from `/docs/README.md`
```

Comment:
On the one hand, this is obsolete. On the other, I too think we should have a `docs/README.md`, that describes our documentation structure.